src/components/calculator/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+
+const Calculator = () => (
+  <div className="wrapper">
+    <div className="output">
+      0
+    </div>
+    <div className="keys">
+      <div className="ac">AC</div>
+      <div className="plus-minus">+/-</div>
+      <div className="percent">%</div>
+      <div className="division">/</div>
```

Comment:
- [ ] For the division operator, you're to use the other sign and not the forward slash. You can use the HTML entity code `&divide;` to achieve it or you copy the sign from the project requirement and past it in your JSX 👍🏽